test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,436 @@ describe('getDefaultAliasIndex', () => {
 		expect(getDefaultAliasIndex(aliasSet)).to.equal(2);
 	});
 });
+
+describe('deleteRelationships', () => {
+	const {Author, Work, Revision, WorkRevision, WorkHeader, Relationship, RelationshipSet, RelationshipType, bookshelf} = orm;
+	let author;
+	let editorJSON;
+
+	beforeEach(async () => {
+		const editor = await createEditor();
+		author = await createAuthor();
+		editorJSON = editor.toJSON();
+	});
+	afterEach(truncateEntities);
+	// let author = await createAuthor();s
+	// const workUUID = getRandomUUID();
+	// const authorBbid = author.get('bbid');
+	// await createRelationshipSet(authorBbid, workUUID, 'Work');
+	// author = await author.refresh({withRelated: 'relationshipSet.relationships'});
+	// const authorJSON = author.toJSON();
+	// const authorRelationships = authorJSON.relationshipSet.relationships;
+	it('it should return an empty array if there are no relationships to delete', async () => {
+		const affectedEntities = await deleteRelationships(orm, null, author);
+		expect(affectedEntities).to.have.length(0);
+	});
+	it('it should return an empty array if the other entity is a deleted entity (no dataId)', async () => {
+		// Create Entities
+		let work = await createWork();
+
+		// Create Relation and its data between two entites
+		const relationshipData = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: work.toJSON().bbid,
+			typeId: 1
+		};
+		const relationshipTypeData = {
+			description: 'test descryption',
+			id: 1,
+			label: 'test label',
+			linkPhrase: 'test phrase',
+			reverseLinkPhrase: 'test reverse link phrase',
+			sourceEntityType: 'Author',
+			targetEntityType: 'Work'
+		};
+		relationshipTypeData.id = random.number();
+		await new RelationshipType(relationshipTypeData)
+			.save(null, {method: 'insert'});
```

Comment:
This can be created once in the `before` function and be reused (store the id in a relationshipTypeId variable)